src/lib.rs

```diff
@@ -164,10 +164,18 @@ fn visit_macros_mut<V: VisitMut>(visitor: &mut V, mac: &mut Macro) {
 impl<'a> VisitMut for FloatLiteralVisitor<'a> {
     fn visit_expr_mut(&mut self, expr: &mut Expr) {
         if let Expr::Lit(lit_expr) = expr {
-            if let Lit::Float(_) = lit_expr.lit {
+            let is_float = match &lit_expr.lit {
+                // Literals like `20f64` are parsed as `LitInt`s
+                Lit::Int(int_lit) => matches!(int_lit.suffix(), "f32" | "f64"),
+                Lit::Float(_) => true,
+                _ => false,
+            };
+
+            if is_float {
                 let mut adapted_replacement = self.replacement.clone();
                 replace_literal(&mut adapted_replacement, self.placeholder, lit_expr);
                 *expr = adapted_replacement;
+                //assert!(false, "expr: {:?}, is float: {}", expr, is_float);
```

Comment:
Maybe kick this one out, or is there a good reason for keeping it...?

Reply:
Oops